asteroidsyncserviced/dbusinterface.cpp

```diff
@@ -29,11 +29,16 @@
 
 DBusWatch::DBusWatch(Watch *watch, WatchesManager* wm, QObject *parent): QObject(parent), m_watch(watch), m_wm(wm)
 {
+  
+    QDBusConnection::sessionBus().registerService("org.asteroidsyncservice");
+    QDBusConnection::sessionBus().registerObject("/org/asteroidsyncservice/Watch", this, QDBusConnection::ExportScriptableSlots|QDBusConnection::ExportScriptableSignals);
```

Comment:
Why do you register every DBusWatch objects as /org/asteroidsyncservice/Watch?
Each time a DBusWatch is created, it should be registered on the bus under "/org/asteroidsyncservice/" + address https://github.com/AsteroidOS/asteroidsyncservice/blob/master/asteroidsyncserviced/dbusinterface.cpp#L131

Reply:
I remove this, thank you for the explanation.